docs/docs/troubleshooting/connection-issues.mdx

```diff
@@ -60,28 +37,18 @@ include:
     shield: settings_reset
 ```
 
-Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
+If you are building for an onboard controller keyboard that doesn't use a shield, you can add a new `shield:` field with its `board:` entry.
 
-</TabItem>
-
-<TabItem value="download">
-1. [Open the `Build` workflow](https://github.com/zmkfirmware/zmk/actions/workflows/build.yml?query=event%3Apush+branch%3Amain+is%3Asuccess) from the `Actions` tab of the ZMK GitHub repository.
-1. Find one of the results for which the `core-coverage` job ran successfully, indicated by a green checkmark in the "core-coverage" bubble like the image example below.
-1. From the next page under "Artifacts", download and unzip the `<board_name>-settings_reset-zmk` zip file for the UF2 image.
-
-| ![Successful core-coverage Job](../../docs/assets/troubleshooting/splitpairing/corecoverage.png) |
-| :----------------------------------------------------------------------------------------------: |
-|    An example of a successful core-coverage job which will produce a settings_reset firmware.    |
+Save the file, commit the changes and push them to GitHub. Download the new firmware zip file build by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
```

Comment:
```suggestion
Save the file, commit the changes and push them to GitHub. Download the new `firmware.zip` file built by the latest GitHub Actions job. In it you will find an additional `settings_reset` UF2 image file.
```

Reply:
I think the zip file won't be literally called `firmware.zip`, hence the generic "the new firmware zip file" phrasing here. We should do "file that is built" rather than "file build".